unittests/utils/test_utils_sanitise.py

```diff
@@ -0,0 +1,45 @@
+from utils.utils_sanitise import sanitise_wildcards, restore_wildcard, sanitise_domain
+from assertpy import assert_that
+
+
+def test_wildcard_replaced_by_random_string():
+    input_dict = {
+        "Name": "\\052.wildcard.celidor.io.",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [{"Value": "neverssl.com"}],
+    }
+    result = sanitise_wildcards(input_dict)
+    prefix = result["Name"].split(".")[0]
+
+    assert_that(len(prefix)).is_equal_to(19)
```

Comment:
This could be simplified to `assert_that(prefix).is_length(19)`

Reply:
good improvement!